### DISCUSSION TITLE
Problema voti non calcolati nella media

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#146] [Richieste di aiuto] [answered]
Buongiorno ,sono qui per riportare una problematica già riscontrata sicuramente nel passato.Quando ho fatto le prove d'ingresso per gli alunni delle classi prime , dovevo toglierle dal calcolo della media ,ossia ,non dovevano fare media ,ma così facendo ,toglieva la possibilità ai genitori di visualizzare la valutazione, quindi ho dovuto fare alunno per alunno affinché i genitori potessero visualizzare il voto che non faceva media, ma , così facendo ,il lavoro si moltiplica

**Comment by trinko** [comment]
Non capisco il problema: se selezioni l'opzione NON IN MEDIA di una valutazione, il genitore continua a vedere il voto (viene mostrato in grigio).

**Comment by manuelbarberi** [comment, marked as answer]
Come dice @trinko i voti che non fanno media dovrebbero essere visibili ai genitori. Se invece inserisci nascosti ai genitori non puoi cambiare l'opzione "fanno media"
